ToDoApp/components/checkBox/CheckBox.js

```diff
@@ -1 +1,26 @@
-//File
+import React, { Component } from 'react';
+import { Image, View, StyleSheet} from 'react-native';
+
+class CheckBox extends Component {
+    render() {
+      return (
+        <View style={{ flex: 1, justifyContent: "flex-end", alignItems: "flex-end", backgroundColor: 'white'}}>
```

Comment:
Please, make a style file separated and put the styles there.

Reply:
I have just changed that, in the next push it will be added.